coalahtml/coala_html.py

```diff
@@ -29,13 +29,34 @@
 from coalib.output.printers.ListLogPrinter import ListLogPrinter
 
 
+def gen_file_set(dirpath):
+    """
+    Parse an output from `os.walk` and create a set of its contents.
+    :param dirpath:
+        Source directory.
+    :return:
+        A set of directory contents traversed recursively.
+    """
+    dir_content = os.walk(os.path.abspath(dirpath))
+    content_set = set()
+    for dir_tuple in dir_content:
+        root, dir_list, file_list = dir_tuple
+        for file_name in [root] + dir_list + file_list:
```

Comment:
Seems like I don't need `os.abspath` as the input is already absolute paths. (from `get_file` and `create_dir`) :)